packages/openapi-ts/src/index.ts

```diff
@@ -184,6 +188,43 @@ const getTypes = (userConfig: UserConfig): Config['types'] => {
   return types;
 };
 
+const getInstalledDependencies = (): Dependencies => {
+  const toReducedDependencies = (p: PackageDependencies): Dependencies =>
+    [p.dependencies ?? {}, p.devDependencies ?? {}].reduce(
+      (deps, devDeps) => ({
+        ...deps,
+        ...devDeps,
+      }),
+      {},
+    );
+
+  let dependencies: Dependencies = {};
+
+  // Attempt to get all globally installed pacakges.
+  const result = sync('npm', ['list', '-g', '--json', '--depth=0']);
```

Comment:
this breaks usage in projects that have configured `npm` to be intercepted by corepack, and then using another package manager.

<img width="1403" alt="image" src="https://github.com/hey-api/openapi-ts/assets/1404810/0be7883d-e225-420b-8246-2bba0a03ead8">

Test by doing `corepack enable && corepack enable npm` and `corepack use yarn@4`

Reply:
Thanks @SimenB, I think I'll just remove dependency checks, there are too many variants to consider given the value of this feature

Reply:
@SimenB this will be removed in v0.45.0, please let me know if that works for you once you try it otu

Reply:
Yeah, 0.45 works. Thanks!